Resources/Prototypes/_LostParadise/Entities/Objects/Weapons/Guns/Ammunition/Projectile/LPPBullets.yml

```
@@ -167,3 +167,30 @@
       types:
         Piercing: 3
         Radiation: 8
+
+- type: entity
+  noSpawn: true
+  parent: BaseBullet
+  id: LPPBulletAntiMaterielNonlethal
+  name: bullet (.60 anti-materiel Non-lethal)
+  components:
+  - type: Projectile
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)